templates/vue/src/services/wp.js

```diff
@@ -0,0 +1,5 @@
+export const data = {
+  ...wpData,
+  apiUrl,
+  postId: wpPostId,
```

Comment:
This might be missing imports.

DuplicateTapestryButton test has a warning:
```
(node:263) UnhandledPromiseRejectionWarning: ReferenceError: wpData is not defined
(Use `node --trace-warnings ...` to show where the warning was created)
(node:263) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:263) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:263) UnhandledPromiseRejectionWarning: ReferenceError: wpData is not defined
(node:263) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
```

Reply:
Thanks, I'll take a look at this.

Reply:
I looked into this and I think it's because I haven't updated all of the places that use `wpData`. It's weird though because the `DuplicateTapestryButton` component doesn't use `wpData` anywhere.

I think it's ok for now but it should be addressed whenever we [refactor the wp module](https://app.asana.com/0/1176098622515689/1195710906078745).